pylutron/__init__.py

```diff
@@ -69,6 +73,7 @@ def __init__(self, host, user, password, recv_callback):
     self._lock = threading.Lock()
     self._connect_cond = threading.Condition(lock=self._lock)
     self._recv_cb = recv_callback
+    self._controller = Controller.UNKNOWN
```

Comment:
What's the value of noting this?  Just in case we encounter a scenario where we need to know?

Maybe at least add a property for retrieving this.

Reply:
done